internal/jobs_test.go

```diff
@@ -191,3 +191,60 @@ func TestAccJobsListAllNoDuplicatesNoTranspile(t *testing.T) {
 	}
 	assert.Equal(t, len(all), len(ids), "Listing produced duplicate results")
 }
+
+func TestAccJobsListWithLimit(t *testing.T) {
```

Comment:
```suggestion
func TestAccJobsListWithLimitNoTranspile(t *testing.T) {
```

to properly work with test cross-compilation infra, as we don't yet support transpiling loops